### PR TITLE
[WIP] Create runtime gRPC proto definition

### PR DESCRIPTION
**Pull request by kchrist-rocketfueldev**

### CODE REVIEW

File: common/types/src/runtime.proto [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    google.protobuf.Any value = 2;
+}
+
+message RuntimeConfig {
+  // Define fields for the runtime configuration
+  // ...
+}
+
+/* getName */
+message GetNameRequest {
```
Comment (harrytou) [inline common/types/src/runtime.proto:19]: Can we be more explicit on what `GetNameRequest` is? Which name does it request?

File: common/types/src/runtime.proto [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  RuntimeConfig config = 1;
+}
+
+message GetNameResponse {
+  string name = 1;
+}
+
+/* getVersion */
+message GetVersionRequest {
```
Comment (harrytou) [inline common/types/src/runtime.proto:28]: Same here, what version are we requesting and of what?

File: common/types/src/runtime.proto [hunk 3]
```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+package kyve.tendermint;
```
Comment (harrytou) [inline common/types/src/runtime.proto:3]: Are you thinking of creating one `.proto` file for every runtime parent class? Meaning all the EVM runtimes will use X `.proto` as a source, all Cosmos runtime will use `Y` as a source?

Is it possible to create just a single `.proto` file for all runtimes?

**Review by harrytou — COMMENTED** [review]
Also @kchrist-rocketfueldev can we get some more comments in there as seen in https://protobuf.dev/programming-guides/proto3/#adding-comments?

File: common/types/src/runtime.proto (hunk 3 above)
Comment (kchrist-rocketfueldev) [reply common/types/src/runtime.proto:3]: it should not be here, it was a leftover from when i did multiple implementations to compare them

File: common/types/src/runtime.proto (hunk 1 above)
Comment (kchrist-rocketfueldev) [reply common/types/src/runtime.proto:19]: would `getRuntimeName` make more sense?

File: common/protocol/src/index.ts [hunk 4]
```diff
@@ -52,6 +52,24 @@ import { SupportedChains } from "@kyvejs/sdk/dist/constants";
 import { storageProviderFactory } from "./reactors/storageProviders";
 import { compressionFactory } from "./reactors/compression";
 import { cacheProviderFactory } from "./reactors/cacheProvider";
+const grpc = require("@grpc/grpc-js");
+var protoLoader = require("@grpc/proto-loader");
+const PROTO_PATH = "./runtiime.proto";
```
Comment (troykessler) [inline common/protocol/src/index.ts:57]: typo: runtiime -> runtime

**Pull request closed without merging**